esmvaltool/diag_scripts/mpqb/mpqb_lineplot_anncyc.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python
+"""Python example diagnostic."""
+import logging
+import os
+from pprint import pformat
+
+import iris
+import matplotlib.pyplot as plt
+from mpqb_utils import get_mpqb_cfg
+
+from esmvaltool.diag_scripts.shared import group_metadata, run_diagnostic
+from esmvaltool.diag_scripts.shared._base import (
+    ProvenanceLogger,
+    get_plot_filename,
+)
+
+logger = logging.getLogger(os.path.basename(__file__))
+
+
+def get_provenance_record(caption):
+    """Create a provenance record describing the diagnostic data and plot."""
+    record = {
+        'caption': caption,
+        'statistics': ['clim'],
+        'domains': ['user'],
+        'plot_type': 'lineplot',
+        'authors': [
+            'mueller_benjamin',
+            'crezee_bas',
+            'hassler_birgit',
+        ],
+        'projects': ['cmug'],
+        'references': [
+            'acknow_project',
+        ],
+    }
+    return record
+
+
+def main(cfg):
+    """Create lineplot."""
+    ylims = [cfg.pop('y0', None), cfg.pop('y1', None)]
```

Comment:
Same comment here: It looks like `y0` and `y1` are possible optional options for this diagnostic which are not listed in the documentation